atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleQueryFactory.java

```diff
@@ -297,15 +297,15 @@ protected FullQuery getRowsColumnRangeFullyLoadedRowsSubQuery(
         return fullQuery;
     }
 
-    private String wrapQueryWithIncludeValue(String wrappedName, String query, boolean includeValue) {
+    private String wrapQueryWithIncludeValue(String name, String query, boolean includeValue) {
         if (!includeValue) {
-            return query;
+            return "/* " + name + " (" + tableName + ") */ " + query;
```

Comment:
This ensures that if includeValue is not set, the sql statement still gets a "name".